pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -76,13 +76,10 @@ func (*OSSMigrations) AddMigration(mg *Migrator) {
 	accesscontrol.AddMigration(mg)
 	addQueryHistoryMigrations(mg)
 
-	if mg.Cfg != nil && mg.Cfg.IsFeatureToggleEnabled != nil {
-		if mg.Cfg.RBACEnabled {
-			accesscontrol.AddTeamMembershipMigrations(mg)
-			accesscontrol.AddDashboardPermissionsMigrator(mg)
-			accesscontrol.AddAlertingPermissionsMigrator(mg)
-		}
-	}
+	accesscontrol.AddTeamMembershipMigrations(mg)
+	accesscontrol.AddDashboardPermissionsMigrator(mg)
+	accesscontrol.AddAlertingPermissionsMigrator(mg)
+
```

Comment:
What if RBAC is explicitly disabled?

Reply:
RBAC is now enabled by default, so RBAC being disabled is very much an edge case. The main reason behind this change (correct me if I'm wrong @Jguer) is that any conditional migrations make potential changes to migration order, which makes migrations way less reliable, so we're trying to avoid that where possible.